scripts/CEE/hs-must-gather/metadata.yaml

```diff
@@ -36,15 +36,68 @@ rbac:
     - verbs:
         - "get"
         - "list"
+      apiGroups:
+        - "hypershift.openshift.io"
+      resources:
+        - "awsendpointservices"
+        - "hostedclusters"
+        - "nodepools"
+        - "hostedcontrolplanes"
+    - verbs:
+        - "get"
+        - "list"
+      apiGroups:
+        - ""
+      resources:
+        - "nodes"
+    - verbs:
+        - "get"
+        - "list"
+      apiGroups:
+        - "infrastructure.cluster.x-k8s.io"
+      resources:
+        - "awsclusters"
+        - "awsmachines"
+        - "awsmachinetemplates"
+        - "kubevirtclusters"
+        - "kubevirtmachinetemplates"
+        - "kubevirtmachines"
+    - verbs:
+        - "get"
+        - "list"
+      apiGroups:
+        - "capi-provider.agent-install.openshift.io"
+      resources:
+        - "agentclusters"
+        - "agentmachinetemplates"
+        - "agentmachines"
+    - verbs:
+        - "get"
+        - "list"
+      apiGroups:
+        - "cluster.x-k8s.io"
+      resources:
+        - "clusters"
+        - "machinedeployments"
+        - "machines"
+        - "machinesets"
+    - verbs:
+        - "get"
+        - "list"
+      apiGroups:
+        - ""
+      resources:
+        - "secrets"
+    - verbs:
+        - "get"
+        - "list"
+        - "create"
       apiGroups:
         - ""
       resources:
-        - "hostedclusters"    
+        - "pods/portforward"
 envs:
   - key: "CLUSTER_NAME"
```

Comment:
I feel `CLUSTER_NAME` can be really ambiguous and few might overlap, instead we could leverage the `CLUSTER_ID` which is a unique entity.

This can be captured via the namespace label: `api.openshift.com/id: ${CLUSTER_ID}`.